### PR TITLE
Ft: io convert via cli

### PR DESCRIPTION
**Pull request by clemens-fricke**
# Overview
Adds the CLI option for converting spline formats into one another.

For example export the spline defined in a json file directly to an SVG graphic.

## Addressed issues
*  No conversion is possible for the io types.

## Showcase
A short / one-liner example to highlight the (new) feature
```
$ splinepy convert -i gus.json -o gus.svg
```

DRAFT:
Currently some issues with the correct export of this example, but only with the GCC compiler?  

## Checklists
* [ ] Documentations are up-to-date.
* [ ] Added example(s)
* [ ] Added test(s)

### CODE REVIEW

**Comment by j042** [comment]
if this is just for svg, can we just name this `splinepy svg -i stuff.json -o smooth_stuff.svg`

File: splinepy/io/json.py [hunk 1]
```diff
@@ -55,6 +55,12 @@ def load(fname):
             _settings.NAME_TO_TYPE[jbz["SplineType"]](**data_dict)
         )
 
+        # Add show options if any
```
Comment (j042) [inline splinepy/io/json.py:58]: this is nice

File: docs/markdown/spline_plotting.md [hunk 2]
```diff
@@ -81,3 +81,16 @@ gus.show(
 This does not need to be only from 2D->3D but can also be from 1D->2D etc.
 
 This is just the beginning of what you can do with the given plotting and creation capabilities in this library. Please look into the [examples folder](https://github.com/tataratat/splinepy/tree/main/examples) to see more capabilities.
+
+
+## Notebook plotting
```
[the remark this answers is not in the document]
Comment (clemens-fricke) [reply docs/markdown/spline_plotting.md:86]: Done

**Review by j042 — APPROVED** [review]
Awesome, thank you!

File: docs/markdown/spline_plotting.md [hunk 3]
```diff
@@ -81,3 +81,16 @@ gus.show(
...
+
+You can also plot your splines inside a notebook. For this you need to change the vedo backend to 'k3d'. To do this you need to add the following lines to the top of the notebook.
```
[the remark this answers is not in the document]
Comment (clemens-fricke) [reply docs/markdown/spline_plotting.md:88]: You can apparently check if it is from a notebook. See https://stackoverflow.com/questions/15411967/how-can-i-check-if-code-is-executed-in-the-ipython-notebook

[the remark this answers is not in the document]
Comment (j042) [reply docs/markdown/spline_plotting.md:88]: pretty cool

**Comment by clemens-fricke** [comment]
> if this is just for svg, can we just name this `splinepy svg -i stuff.json -o smooth_stuff.svg`

How important is this to you? In the background, I would only add a check that the export file extension  is actually `svg` before going into the same convert function.

**Comment by j042** [comment]
> > if this is just for svg, can we just name this `splinepy svg -i stuff.json -o smooth_stuff.svg`
> 
> How important is this to you? In the background, I would only add a check that the export file extension is actually `svg` before going into the same convert function.

I think it's okay as it is. export functions throws error if they can't understand the input. Thanks for this PR!

**Pull request merged**